common/src/payload.rs

```diff
@@ -0,0 +1,9 @@
+//! This file provides utility functions to initialize an empty payload for a JSON response.
+//!
+//! author: https://github.com/vincenzopalazzo
+use serde_json::json;
+
+/// return a server::json_value useful to initialize the JSON payload as an empty JSON object.
+pub fn init_payload() -> serde_json::Value {
```

Comment:
better keep the name of the file `json_utils` but I think you can just copy and paste it?

Reply:
You are right! Fixed.
Thanks!